### DISCUSSION TITLE
How to have shared options

### DISCUSSION THREAD

**Discussion by JohnScottUK** [jmcdo29/nest-commander#934] [Q&A]
I have a cli tool that takes multiple commands - like git. How would I define the common options:

my-cli [common-options] command [command-options]
e.g.
my-cli --verbose cmdA --optionA ABCD
my-cli --verbose cmdB --optionB BCDE

I can see how to define command specific options against each command (i.e. optionA and optionB), but how to I create common options like --verbose?

In my specific case, I wanted to have --verbose change the logging level to verbose and --debug change the logging level to debugging.

**Comment by jmcdo29** [comment]
I believe [this discussion](https://github.com/jmcdo29/nest-commander/discussions/888) and [this issue](https://github.com/jmcdo29/nest-commander/issues/663) will help determine how this can work
